### PR TITLE
EAP 7.1 integration tests

### PR DESCRIPTION
**Pull request by douglaspalmer**

### CODE REVIEW

File: eap/integration/eap7/src/test/resources/arquillian.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@
       environment, e.g. service host, port, etc. testrunner relates to the pod
       name in testrunner-pod.json.
     -->
-    <container qualifier="testrunner-eap7" mode="suite" default="true">
+    <container qualifier="testrunner-eap7" mode="class" default="true">
```
Comment (jwendell) [inline eap/integration/eap7/src/test/resources/arquillian.xml]: I'm curious why this change was needed...

Comment (douglaspalmer) [reply eap/integration/eap7/src/test/resources/arquillian.xml]: This change makes the tests more stable.

File: eap/integration/eap7/src/test/resources/eap71.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+              ]
+            }
+          }
+        },
+        "image": "docker-registry.engineering.redhat.com/dpalmer/eap71-openshift:1.0-TP",
```
Comment (jwendell) [inline eap/integration/eap7/src/test/resources/eap71.json]: I believe this should be `jboss-eap-7-tech-preview/eap71-openshift:1.0`
If we need to use an unreleased image we can tweak this downstream in our testsuite CI.

File: eap/integration/eap7/src/test/resources/logging.properties [hunk 3]
```diff
@@ -1,13 +1,18 @@
-#handlers = java.util.logging.FileHandler, java.util.logging.ConsoleHandler
-.level = FINEST
+loggers=org.xnio,io.undertow,org.apache,org.jboss,org.wildfly
 
-# Formater
-java.util.logging.SimpleFormatter.format=%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS.%1$tL %4$-7s (%2$s) %5$s %6$s%n
+# Root logger configuration
+logger.level=DEBUG
```
Comment (jwendell) [inline eap/integration/eap7/src/test/resources/logging.properties]: Do we really need DEBUG level?

File: pom.xml [hunk 4]
```diff
@@ -54,6 +53,7 @@
         <version.syslog4j>0.9.30</version.syslog4j>
         <version.json-simple>1.1.1</version.json-simple>
         <version.undertow>1.3.31.Final</version.undertow>
+        <version.io.undertow>1.3.30.Final</version.io.undertow>
```
Comment (jwendell) [inline pom.xml]: Can't we use 1.3.31 as in other places? It's a minor bug fix release that fixes some recursion loops bugs...
